From: Johannes Thumshirn <jthumshirn@suse.de>
Subject: KABI protect structy ata_host
Date: Mon Aug  6 11:04:01 CEST 2018
Patch-mainline: Never, KABI fix
References: git-fixes

Patch patches.fixes/libata-add-refcounting-to-ata_host.patch
introduced a kref into struct ata_host. Guard it with a ifndef
__GENKSYMS__ make the checker happy.

Signed-off-by: Johannes Thumshirn <jthumshirn@suse.de>
---
 include/linux/libata.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/linux/libata.h
+++ b/include/linux/libata.h
@@ -616,7 +616,9 @@ struct ata_host {
 	void			*private_data;
 	struct ata_port_operations *ops;
 	unsigned long		flags;
+#ifndef __GENKSYMS__
 	struct kref		kref;
+#endif
 
 	struct mutex		eh_mutex;
 	struct task_struct	*eh_owner;
